src/commands/indexer/index.ts

```diff
@@ -463,9 +479,79 @@
               // )
 
               await sqsHandleMintEvent.call(this, this.networkMonitor, transaction, job.network, tags)
+            } else if (functionSig === functionSignature('transfer(address,uint256)')) {
```

Comment:
So thsi and the other else seem to do the exact same thing. Could this else if be an or of `transfer` and `fulfillBasicOrder_efficient_6GL6yc` signatures?

Reply:
yeah good call